src/containers/repo.jsx

```diff
@@ -2937,6 +2944,8 @@ class Repo extends Component {
               isMaskMapLoading={this.state.isMaskMapLoading}
               invalidMaskMap={this.state.invalidMaskMap}
               toggleTextTab={this.toggleTextTab}
+              maskOpacity={this.state.maskOpacity}
+              handleOpacityChange={this.handleOpacityChange}
```

Comment:
Here we pass both the state and the state update method down to the child component